index.html

```diff
@@ -0,0 +1,354 @@
+              fill="#505F79"
+            />
+          </svg>
+          <svg
+            width="20"
+            height="20"
+            viewBox="0 0 20 20"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path
+              d="M19.7665 0.369643C19.4331 -0.0290272 18.8665 -0.128695 18.4331 0.170308L16.4998 1.49921C15.5998 0.535755 14.3331 0.00419528 12.9665 0.00419528C10.2665 0.00419528 8.03313 2.19688 8.03313 4.92113C8.03313 5.3198 8.06646 5.68524 8.16646 6.05069C5.83313 5.55235 4.43313 3.85801 3.8998 0.801535C3.83313 0.402865 3.53313 0.103863 3.13313 0.00419528C2.76646 -0.0954722 2.36646 0.0706403 2.13313 0.369643C2.06646 0.436088 -3.43354 8.07726 4.53313 14.5557C3.76646 16.1836 1.46646 17.5789 0.566463 18.044C0.13313 18.2434 -0.06687 18.7085 0.03313 19.1736C0.13313 19.6387 0.53313 19.9377 0.999797 19.9377H2.73313C12.8665 19.9377 17.4665 14.9211 17.5665 3.69189L19.6665 1.69854C20.0665 1.36632 20.0998 0.768313 19.7665 0.369643Z"
+              fill="#505F79"
+            />
+          </svg>
+          <svg
+            width="20"
+            height="16"
+            viewBox="0 0 20 16"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path
+              d="M18.4042 1.88042L20 0.337684V0H14.4717L10.5317 9.90989L6.04917 0H0.2525V0.337684L2.11667 2.60547C2.29833 2.77305 2.39333 3.01811 2.36917 3.26484V12.1768C2.42667 12.4977 2.32333 12.8278 2.1 13.0611L0 15.6328V15.9663H5.95417V15.6286L3.85417 13.0611C3.62667 12.8269 3.51917 12.5027 3.565 12.1768V4.46821L8.79167 15.9705H9.39917L13.8933 4.46821V13.6312C13.8933 13.8728 13.8933 13.9225 13.7367 14.0808L12.12 15.6615V16H19.9633V15.6623L18.405 14.1204C18.2683 14.016 18.1975 13.8408 18.2267 13.6707V2.33011C18.1975 2.15916 18.2675 1.984 18.4042 1.88042Z"
+              fill="#505F79"
+            />
+          </svg>
+        </div>
+        <button class="show">Get my resume</button>
+
+        <ul class="about-cat">
+          <li>Languages</li>
```

Comment:
Your about section is looking really good✨. But please add the arrow icons next to each technology category as shown in the figma design.

Your design:
<img width="198" alt="image" src="https://user-images.githubusercontent.com/37116322/227598022-65a88a65-bfd2-4016-9fd9-5f9aa0b02567.png">

Figma design:
<img width="222" alt="image" src="https://user-images.githubusercontent.com/37116322/227598098-9203731a-4da0-48f2-987a-f0e273435758.png">